src/installation/live-images/partitions.md

```diff
@@ -19,9 +19,12 @@ The following sections will detail the options for partition configuration.
 ## BIOS system notes
 
 It is recommended that you create an MBR partition table if you are using a BIOS
-boot system. This will limit the number of partitions you create to four. It is
-possible to install a GPT partition table on a BIOS system, but grub will need a
-special partition to boot properly.
+boot system. This will limit the number of partitions you create to four.
+
+It is possible to use a GPT partition table on a BIOS system, but grub will
+require a special partition to boot properly. This partition should be at the
+beginning of your disk and have a size of 1MB, with type BIOS boot. Don't create
```

Comment:
```
It is possible to use a GPT partition table on a BIOS system, but GRUB will
require a special partition to boot properly. This partition should be at the
beginning of your disk and have a size of 1MB, with type `BIOS boot`. Don't create
```

Reply:
I think there may be a few other places where "GRUB" needs to be capitalized. Should I do a global search-and-replace for that?

Reply:
If you'd like to do it, yes, but do so in a separate PR. And there _are_ cases where "grub" might be correct, specially if it's inside a code block.